drake_ros_systems/README.md

```diff
@@ -0,0 +1,62 @@
+# Drake ROS Systems
```

Comment:
If reviewing, recommend skipping the README for now. I haven't updated it since I got the example working.

Reply:
The README is now useful :tada: